index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>My Portfolio Website</title>
```

Comment:
- [ ] The page titles you provided have room for improvement. Here are revised versions that better describe the content of the page:
  - Poor titles:

      - Microverse project | Your portfolio.
      - Portifolio project.

  - Better page titles:

      - Mark's Portfolio | Full-Stack Web Developer.
      - Mark's Professional Portfolio | Full-Stack Web Development.

By including your name and specifying your role as a full-stack web developer, the revised titles provide clearer and more informative descriptions of the page content. Kindly implement this.

Reply:
> - [ ] In response to the previous code reviewer's request, it is suggested that you make improvements to the page title you provided. The revised version mentioned above will offer clearer and more informative descriptions of the content on the page. Using this updated title will enhance accessibility and improve the understanding of the page's purpose.☺